Rally/app/src/main/java/com/example/compose/rally/ui/overview/OverviewScreen.kt

```diff
@@ -128,7 +130,9 @@ private fun AlertHeader(onClickSeeAll: () -> Unit) {
 @Composable
 private fun AlertItem(message: String) {
     Row(
-        modifier = Modifier.padding(RallyDefaultPadding),
+        modifier = Modifier
+            .padding(RallyDefaultPadding)
+            .semantics(mergeDescendants = true) {},
```

Comment:
Add a comment explaining, also about the difference with  `Modifier.clearAndSetSemantics {}`

Reply:
There's barely any comments in this sample as it is. I'm adding these same properties to all sample apps. Is it really desired to add a comment "Merges the semantics properties of the descendants" to each of these? Or would it be best to have good documentation on these topics and leave it like this.

Reply:
The comment shouldn't say what it does but why it does it in this case.

> There's barely any comments in this sample as it is. 
Not an excuse :)